web/packages/extension/assets/player.html

```diff
@@ -76,6 +76,10 @@
                 </div>
             </div>
         </div>
+        <dialog id="modal">
+            <button id="grant" type="button">Grant permissions</button>
+            <button id="close" type="button">Close</button>
```

Comment:
I forgot i18n for this, I'll get to that later.

Reply:
Actually, nevermind, just remembered we should save that for https://github.com/ruffle-rs/ruffle/pull/14628 once we determine what to do about the demo page too.